client/auto_update.cpp

```diff
@@ -15,6 +15,8 @@
 // You should have received a copy of the GNU Lesser General Public License
 // along with BOINC.  If not, see <http://www.gnu.org/licenses/>.
 
+DEPRECATED - we're not going to do auto-update
```

Comment:
Maybe it's still better to make this line a proper comment?

Reply:
This is what I do with deprecated files.
They're not supposed to compile, and this way people will see that immediately.